diff --git a/config/env.js b/config/env.js
index 4d134a0aa07c5534ad87af073b341858ad9ebec4..bc75e88e303112980d4cff937c40376a586d464e 100644
--- a/config/env.js
+++ b/config/env.js
@@ -66,7 +66,7 @@ process.env.NODE_PATH = (process.env.NODE_PATH || '')
 
 // Grab NODE_ENV and REACT_APP_* environment variables and prepare them to be
 // injected into the application via DefinePlugin in webpack configuration.
-const REACT_APP = /^REACT_APP_/i;
+const REACT_APP = /^REACT_APP_|TARO_|LIBRARY_/i;
 
 function getClientEnvironment(publicUrl) {
   const raw = Object.keys(process.env)